assignment-1/src/task4.py

```diff
@@ -1,97 +0,0 @@
-import time
-import contextlib
-import inspect
-import atexit
-import traceback
-import datetime
-import io
-
-def exit_hd():
-    """To print the ranks of each program just before the program is finished"""
-    sorted_ranks = sorted(ranks.items(), key=lambda x: x[1])
-    if len(sorted_ranks) > 0:
-        print("\nPROGRAM\t|\tRANK\t|\tTIME ELAPSED", end="")
-        for i in range(len(sorted_ranks)):
-            print("\n", sorted_ranks[i][0], "\t\t", i + 1, "\t\t", sorted_ranks[i][1], "sec", end="")
-atexit.register(exit_hd)
-
-
-ranks = dict()  # To store the execution time, and name of the functions
-sorted_ranks = []
-class decorator_4:
```

Comment:
Class names should have CamelCase style.

Reply:
Noted!